ledger/src/blocktree_db.rs

```diff
@@ -248,6 +261,36 @@ pub trait TypedColumn: Column {
     type Type: Serialize + DeserializeOwned;
 }
 
+impl TypedColumn for columns::TransactionStatus {
+    type Type = (Vec<u8>, u64);
```

Comment:
Suggest:
```
    type Type = (transaction::Result<()>, u64);
```
Since you've implemented a TypedColumn already, this saves all the hand-rolled serialization.